gen/calc.py

```diff
@@ -557,7 +557,8 @@ def calculate_set(parameter):
         'minuteman_forward_metrics': 'false',
         'minuteman_min_named_ip_erltuple': calculate_minuteman_min_named_ip_erltuple,
         'minuteman_max_named_ip_erltuple': calculate_minuteman_max_named_ip_erltuple,
-        'mesos_isolation': calculate_mesos_isolation,
+        'mesos_isolation': 'cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,docker/runtime,docker/volume'
```

Comment:
Looks like something moved in underneath me in a rebase somewhere. @nickbp @cmaloney Looks like the method returns one set of isolators and this is appending the metrics isolator. Do you think I can simply append the metrics isolator to what is being returned in the method, similarly to what we do here?